napari/layers/vectors/vectors.py

```diff
@@ -243,7 +243,8 @@ def data(self, vectors: np.ndarray):
         self._displayed_stored = copy(self.dims.displayed)
 
         self._update_dims()
-        self.events.data()
+        self._update_editable()
```

Comment:
What's editable exactly? I noticed @tlambert03 asking about it in another comment but to be honest the answer didn't really clarify anything for me...?

Reply:
Editable is both a user setting and an attribute automatically determined that sets where things like adding points/ drawing with the paintbrush etc are allowed or not. We added both because someone might want to disable such functionality themselves AND because there are certain configurations like multiscale or 3D viewing where we don't support such interactivity. Definitely room for improvement here, but that's what the parameter is